llvm/test/MC/Disassembler/SystemZ/insns.txt

```diff
@@ -10656,18 +10752,6 @@
 # CHECK: lxdbr %f13, %f15
 0xb3 0x05 0x00 0xdf
 
-# CHECK: lxdr %f0, %f8
```

Comment:
Why are you removing these?  That isn't mentioned in the commit message at all ...

Reply:
my apologies, it seems i messed up the commits i aggregated into this PR. I believe i have now fixed the issues mentioned.